docs/decisions/0018-expose-all-sass-vars-as-css-vars.rst

```diff
@@ -0,0 +1,75 @@
+0018 CSS Variables
+##################
+
+
+Status
+******
+
+Draft
+
+Context
+*******
+
+.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+All Open edX MFEs use Bootstrap via Paragon for styling. Each MFE builds its
+own styling which is a combination of common styles from Bootstrap + Paragon,
+on which it addds its own SCSS.
+
+If we want to be able to support a common pre-built core stylesheet that can be
```

Comment:
```suggestion
If there is a need to support a common pre-built core stylesheet that can be
```

Reply:
I'm assuming in this ADR that the need is there, so I think I'm happy with the original phrasing.